src/reference/01-General-Info/05-Using-Sonatype.md

```diff
@@ -2,26 +2,89 @@
 out: Using-Sonatype.html
 ---
 
+  [sonatype-ossrhguide]: http://central.sonatype.org/pages/ossrh-guide.html
+  [sonatype-signup]: https://issues.sonatype.org/secure/Signup!default.jspa
+  [sonatype-new-project]: https://issues.sonatype.org/secure/CreateIssue.jspa?issuetype=21&pid=10134
+  [sonatype-requirements]: http://central.sonatype.org/pages/requirements.html
+  [sonatype-coordinates]: http://central.sonatype.org/pages/choosing-your-coordinates.html
+  [sonatype-nexus]: https://oss.sonatype.org/#welcome
+  [sonatype-pgp]: http://central.sonatype.org/pages/working-with-pgp-signatures.html
   [sbt-pgp]: http://scala-sbt.org/sbt-pgp/
+  [sbt-sonatype]: https://github.com/xerial/sbt-sonatype
+  [sbt-release]: https://github.com/sbt/sbt-release
+  [gnupg]: https://www.gnupg.org/
 
-Deploying to Sonatype
+Using Sonatype
 ---------------------
-
 Deploying to sonatype is easy! Just follow these simple steps:
 
+## Sonatype setup
+> *Note:* The reference process for configuring and publishing to Sonatype is 
+described in their [OSSRH Guide][sonatype-ossrhguide].
+ 
+In short, you need two publicly available URLs: 
+* the website of the project e.g. https://github.com/sonatype/nexus-oss
+* the project's source code e.g. https://github.com/sonatype/nexus-oss.git
+
+The [OSSRH Guide][sonatype-ossrhguide] walks you through the required 
+process of setting up the account with Sonatype. It’s as simple as 
+[creating a Sonatype's JIRA account][sonatype-signup] and afterwards creating a 
+[New Project ticket][sonatype-new-project]. When creating the account try to 
+use the same domain in your email address as the project is hosted on. I guess 
+it makes it easier to validate the relationship with the groupId requested in 
+the ticket, but is not the only method used to confirm the ownership. 
+
+Creation of the New Project ticket is as simple as:
+* providing the name of the library in the ticket’s subject,
+* naming the groupId you want to use for distributing the library (make sure 
+it is matching the root package of your code). Sonatype provides you with 
+additional hints on choosing the right groupId for publishing your library in 
+[Choosing your coordinates guide][sonatype-coordinates].
+* providing the SCM and Project URLs to the source code and homepage of the 
+library.
+
+> *Note:* After creating you Sonatype account (on their JIRA) you can login 
+using the same credentials to the [Nexus Repository Manager][sonatype-nexus] 
+which is not required to be used in this guide, but can be used later to check 
+on the published artifacts.
+
+Notice that Sonatype advises that responding to the New Project ticket might 
+take up to two business days, but in my case it was few minutes.
+
+## SBT setup
```

Comment:
Happens again here where you see an extra page heading in the middle of the page.

![screen shot 2017-03-30 at 1 24 51 am](https://cloud.githubusercontent.com/assets/358615/24488843/bf22979c-14e7-11e7-88f1-5c6b638860ec.png)